generators/client/files.js

```diff
@@ -0,0 +1,151 @@
+const jhipsterConstants = require('generator-jhipster/generators/generator-constants');
+
+const ANGULAR_DIR = jhipsterConstants.ANGULAR_DIR;
+const REACT_DIR = jhipsterConstants.REACT_DIR;
+
+const CLIENT_TEST_SRC_DIR = jhipsterConstants.CLIENT_TEST_SRC_DIR;
+
+const clientFiles = {
+    angularMain: [
+        {
+            condition: generator => generator.clientFramework === 'angularX',
```

Comment:
The files object is exported from the base-generator we can use that here rather than re-writing the files here

Reply:
I'm not sure how you mean, can you elaborate?